### DISCUSSION TITLE
DoubleRowLayout.qml一个页面内使用两个，不关闭页面切换时，第二个组件initSplitterX不对

### DISCUSSION THREAD

**Discussion by PinelliaLogic** [hiroi-sora/Umi-OCR#461] [General]
![1](https://github.com/hiroi-sora/Umi-OCR/assets/95219163/5e003953-bd47-492b-887b-6dbca5e94dee)
![2](https://github.com/hiroi-sora/Umi-OCR/assets/95219163/7184d3e8-f5a7-46e7-a91e-001643049187)

**Comment by hiroi-sora** [comment]
https://github.com/hiroi-sora/Umi-OCR/issues/464

Bug已修复。

你可以修改：

`UmiOCR-data\qt_res\qml\Widgets\DoubleRowLayout.qml` 拉到最下，246行，隐藏容器的宽度，改为：
```
            width: Math.max(doubleCC.width, 400)
```

同理， `UmiOCR-data\qt_res\qml\Widgets\DoubleColumnLayout.qml` 191行的高度，改为：
```
            height: Math.max(doubleCC.height, 400)
```

**Comment by PinelliaLogic** [comment]
感谢大佬👍

**Comment by hiroi-sora** [comment]
顺便，如果你不想让内层 Layout 小一圈，希望所有子面板的高度相同。可以同步 [这个更新](https://github.com/hiroi-sora/Umi-OCR/commit/37981d985dc179bdd563c169cc331189063b49c1) ，然后设置内层 DoubleRowLayout 的 `margins: 0`

![image](https://github.com/hiroi-sora/Umi-OCR/assets/56373419/b10849f1-d715-4414-9c29-be8c7ba3d542)
